calico_kubernetes/calico_kubernetes.py

```diff
@@ -2,18 +2,23 @@
 import json
 import os
 import sys
+import re
+import socket
 from docker import Client
 from docker.errors import APIError
 from subprocess import check_output, CalledProcessError, check_call
 import requests
-from urllib import quote
 import sh
-import re
+from netaddr import IPAddress, AddrFormatError
+from pycalico import netns
 from pycalico.datastore import IF_PREFIX, DatastoreClient
 from pycalico.util import generate_cali_interface_name, get_host_ips
-from pycalico.datastore_datatypes import Rules
+from pycalico.ipam import IPAMClient
+from pycalico.datastore_errors import PoolNotFound
 
 DOCKER_VERSION = "1.16"
+ORCHESTRATOR_ID = "docker"
+hostname = socket.gethostname()
```

Comment:
hostname --> HOSTNAME

All caps makes it more clear that this is a global.

Alternatively, make it "self.hostname" and add it to NetworkPlugin.__init__